h2o-py/h2o/utils/threading.py

```diff
@@ -0,0 +1,74 @@
+from __future__ import absolute_import
+
+import contextlib
+import threading
+
+try:
+    # not ideal, but as we're going to add various keys to this local context,
+    # then simply initialize it as a dictionary that we're going to manipulate.
+    # This also allows a simplified shared interface (dict) for context vars and thread-locals.
+    from contextvars import ContextVar, copy_context  # Py 3.7+
+    local = ContextVar('h2o_context')
+    local.set({})
+    
+    def _get_local(copy=True):
+        loc = local.get()
```

Comment:
I've no idea how to reproduce this.
I made it more robust by setting a default value to the `ContextVar` instead of setting an initial value... should help in those weird situations.